src/search3_httpd.erl

```diff
@@ -177,6 +197,20 @@ parse_positive_int_param(Name, Val, Prop, Default) ->
         throw({query_parse_error, ?l2b(Msg)})
     end.
 
+parse_positive_int_param2(Name, Val) ->
```

Comment:
can we improve this name? Perhaps we need a function that can handle valid ranges, and then call it with different values? (e.g, with list_to_integer(config:get("search3", Prop, Default) as one of the arguments for the first case)